Makefile

```diff
@@ -187,16 +195,35 @@ build-linux:
 
 # Run a 4-node testnet locally
 docker-start:
-	@echo "Wait until 'Attaching to node0, node1, node2, node3' message appears"
-	@if ! [ -f build/node0/config/genesis.json ]; then docker run --rm -v `pwd`/build:/tendermint:Z tendermint/localnode testnet --v 4 --o . --populate-persistent-peers --starting-ip-address 192.167.10.2 ; fi
+	@if ! [ -f build/node0/config/genesis.json ]; then docker run --rm -v $(CURDIR)/build:/tendermint:Z tendermint/localnode testnet --v 4 --o . --populate-persistent-peers --starting-ip-address 192.167.10.2 ; fi
 	docker-compose up
 
 # Stop testnet
 docker-stop:
 	docker-compose down
 
+###########################################################
+### Remote full-nodes (sentry) using terraform and ansible
+
+# Server management
+server-setup:
+	@if [ -z "$(DO_API_TOKEN)" ]; then echo "DO_API_TOKEN environment variable not set." ; false ; fi
+	@if ! [ -f $(HOME)/.ssh/id_rsa.pub ]; then ssh-keygen ; fi
+	cd networks/remote/terraform && terraform init && terraform apply -var DO_API_TOKEN="$(DO_API_TOKEN)" -var SSH_KEY_FILE="$(HOME)/.ssh/id_rsa.pub"
+	@if ! [ -f $(CURDIR)/build/node0/config/genesis.json ]; then docker run --rm -v $(CURDIR)/build:/tendermint:Z tendermint/localnode testnet --v 0 --n 4 --o . ; fi
+	cd networks/remote/ansible && ANSIBLE_HOST_KEY_CHECKING=False ansible-playbook -i inventory/digital_ocean.py -l remotenet install.yml
+	@echo "Next step: Add your validator setup in the genesis.json and config.tml files and run \"make server-config\". (Public key of validator, chain ID, peer IP and node ID.)"
```

Comment:
and where do I get my validator setup?

Reply:
The use-case is that validators want to set up sentry nodes for their testnet. They should already have a validator running somewhere on their other server. Alternatively, they can run `tendermint init` on another server which creates a validator configuration for testing. They can copy the files over from their.

Reply:
Maybe the targets should be renames from server-setup and such to sentry-setup... What do you think?

Reply:
`remotenet-start`. btw, why sentry and not plain network like local? how can I setup remote network without complicated sentry arch?

Reply:
You have to define "remote network" better. We decided that the only use-case for remote network is the sentry-node setup - which is a number of full nodes connecting to a previously setup validator node. There's no use-case yet for a set of validators in the cloud managed by one entity. The decision is so far, that if you want to be a validator, you build and deploy yourself.

Reply:
> from server-setup and such to sentry-setup... What do you think?

let's rename to sentry-* .. (sentry-start, ...). because if we leave `remotenet`, people will think it works exactly like `localnet` but in a cloud, which is not true.